### PR TITLE
restructuring, web3, paris, pages, etc - we are schroot

### PR DESCRIPTION
**Pull request by brock-haugen**
* switched UI to typescript
* added initial home, personal gallery, and gift pool pages
* enabled SvgSketch to be imported with "static" seed
* hooked up rudimentary `RequireWeb3` provider
* included `@bridgetown-collective/paris` for fetching NFT data

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,18 +1,4 @@
...
-       "minify": true,
-       "transpileTemplateLiterals": true,
-       "pure": true,
-       "displayName": true,
-       "preprocess": false
-     }
-   ]
- ]
+  "presets": ["next/babel"],
+  "plugins": [["styled-components", { "ssr": true }]]
```
Comment (r0mille) [inline .babelrc:3]: nice - i think i had a lot of copy pasta from other projects without really understanding what i needed

File: components/about.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+
+export default function About() {
+  return (
+    <div>
+      <p className="text-2xl m-2">Information about the project coming soon</p>
```
Comment (r0mille) [inline components/about.tsx:6]: oh so you're down with tailwind?

File: components/about.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+      <Link href="/the-pool">
+        <button>Dive into the Gift Pool</button>
```
Comment (r0mille) [inline components/about.tsx:8]: oo a swimming pool of wrapped gifts

File: components/image_generation/spaceFiller.tsx [hunk 4]
```diff
@@ -3,74 +3,76 @@ import RND from "./randomizer";
 const canvasWidth = 478;
 const canvasHeight = 478;
 
-const c = 1.1 
+const c = 1.1;
```
Comment (r0mille) [inline components/image_generation/spaceFiller.tsx:6]: haha this file was leftover from another project - it tries to do randomly place objects without them overlapping - i can remove it

File: components/about.tsx (hunk 2 above)
Comment (brock-haugen) [reply components/about.tsx:6]: haha I just used it because you had already included it 😅 no real preference on my end

File: .babelrc (hunk 1 above)
Comment (brock-haugen) [reply .babelrc:3]: Same 😂 I think this is effectively the same, but seemed to help some issue I was seeing

File: pages/the-pool.tsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { NFTsByOwner } from "@bridgetown-collective/paris";
+
+// @TODO: set post deployment - maybe as envvar?
+const CONTRACT_ADDRESS = "";
```
Comment (r0mille) [inline pages/the-pool.tsx:4]: envvar works  for me

**Review by r0mille — APPROVED** [review]

**Pull request merged**